lib/src/svg/theme.dart

```diff
@@ -22,16 +23,23 @@ class SvgTheme {
   /// See: https://www.w3.org/TR/SVG11/coords.html#Units
   final double fontSize;
 
+  /// The x-height (corpus size) of the font used when calculating ex units of SVG elements.
+  /// Defaults to [fontSize] / 2 if not provided.
+  /// See: https://www.w3.org/TR/SVG11/coords.html#Units, https://en.wikipedia.org/wiki/X-height
+  final double xHeight;
+
   @override
   bool operator ==(dynamic other) {
-    if (other.runtimeType != runtimeType) {
-      return false;
+    if (identical(this, other)) {
```

Comment:
What's the reason for this change?

This class shouldn't really be expensive enough to compare yet to benefit from this.

Reply:
This method should not be expensive as it would only check whether two references are the same, right? The reason why I wanted to avoid `runtimeType` is that I found `runtimeType.toString` may have different values under minification and saw some recommendations to avoid it ([see here](https://github.com/dart-lang/sdk/issues/35052#issuecomment-621644698)). Let me know if you prefer to keep `runtimeType` here.

Reply:
Yes, runtimeType toString should be avoided but this does not use that.  Please change it back

Reply:
Sure, reverted.